src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -276,13 +302,26 @@ export class RelayFeeCalculator {
   async relayerFeeDetails(
     amountToRelay: BigNumberish,
     tokenSymbol: string,
-    tokenPrice?: number,
-    _originRoute?: ChainIdAsString,
-    _destinationRoute?: ChainIdAsString
+    originRoute: ChainIdAsString,
+    destinationRoute: ChainIdAsString,
+    messagePayload?: {
+      message: string;
+      recipientAddress: string;
+      relayerAddress?: string;
```

Comment:
This is a little weird since relayer address isn't really part of the message payload. Does it need to be coupled to the other two? Seems like it could just be a separate optional arg (as said above, I think all these should just be decomposed anyway).

Reply:
I agree with Matt's instinct that these shouldn't be coupled

Reply:
This has been uncoupled via https://github.com/across-protocol/sdk-v2/pull/427/commits/d94450934f46b17d6fdd0a9ce687378490b4bb64